CondTools/SiPixel/plugins/SealModules.cc

```diff
@@ -0,0 +1,17 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
```

Comment:
Since you are modernizing, you could further define all these `FWK_MODULE`'s inside their own implementation file.
Even better if you could also merge the header and implementation files in \plugin: they all seem rather easy for doing so

Reply:
Ok, I'll merge the headers